src/App.js

```diff
@@ -124,30 +120,27 @@ function LoyaltyBar({max, current, change}) {
     changeClass += ' Heal';
     base -= change;
   }
-  return <div className={baseClass}>
-    <div className="LoyaltyBarInside" style={{width: base + '%'}}/>
-    <div className={changeClass} style={{width: change + '%'}}/></div>;
+  return (
+    <div className={baseClass}>
+      <div className="LoyaltyBarInside" style={{ width: base + '%' }} />
+      <div className={changeClass} style={{ width: change + '%' }} />
+    </div>
+  );
 }
 
 function SimpleAttack(props) {
-  //const [ref, api] = useSphere(() => ({type: 'Static', radius: 0.2}));
   const mesh = useRef();
   const sourceHero = props.sourceHero.current;
   const targetHero = props.targetHero.current;
   useFrame(() => {
     const phase = props.turnClock.phase;
-    const sX = sourceHero.position.x;
-    const sY = sourceHero.position.y;
-    const tX = targetHero.position.x;
-    const tY = targetHero.position.y;
-    const aX = sX * (1 - phase) + tX * phase;
-    const aY = sY * (1 - phase) + tY * phase;
-    mesh.current && mesh.current.position.set(aX, aY, 2);
+    const src = sourceHero.position;
+    const dst = targetHero.position;
+    mesh.current && mesh.current.position.lerpVectors(src, dst, phase);
     if (props.turnClock.time === turnFrames && props.attackTurn === 0) {
-      const force = new THREE.Vector3(tX - sX, tY - sY, 0);
-      force.normalize().multiplyScalar(100);
+      const force = dst.clone().sub(src).normalize().multiplyScalar(5);
       const localForce = targetHero.worldToLocal(force);
-      targetHero.physicsApi.applyLocalForce(localForce.toArray(), [0, 0, 1]);
+      targetHero.physicsApi.applyLocalImpulse(localForce.toArray(), [0, 0, 1]);
```

Comment:
Why did you change this? I think force is better because then heavier heros are affected less. What is the benefit of impulse?